### ISSUE TITLE
Combine apt-get RUN lines and clean up afterward

### ISSUE THREAD

**Issue by mtmorgan** [reed-mcdaniel-716/bioc_hub_server_docker#1]
These lines update the apt database and install system dependencies

https://github.com/reed-mcdaniel-716/bioc_hub_server_docker/blob/62f02f5997476981ca591525cc484d35d1f22278/Dockerfile#L35-L45

Each RUN creates a separate layer, and once a layer is created (the RUN statement ends) the docker image always contains that layer -- it might be 'masked' in a subsequent command, but it's always there. This means that the database of packages available for installation, for example, are always present on the docker image, even though apt will not be used again during the docker build process or when the image is running.

A better practice is (I think!) to have a single RUN statement for all operations, and to clean up as a final part of the RUN. A slightly complicated example starts at https://github.com/Bioconductor/bioconductor_docker/blob/a8a077ab06df777ea6a32cbc69b69cac759645f3/Dockerfile#L33-L35 and continues installing various dependencies until done, and then cleans up at https://github.com/Bioconductor/bioconductor_docker/blob/a8a077ab06df777ea6a32cbc69b69cac759645f3/Dockerfile#L121-L122

**Comment by reed-mcdaniel-716** [comment]
I agree, we have a similar approach in our registry apps. I will look into the specific clean-up needed for the specified installs
